jp/stripStopWords.js

```diff
@@ -0,0 +1,26 @@
+module.exports={
+	strip_stopwords: function(languageCode, inputText){
+		var fileSystem = require('fs');
+		var validLanguageCodes = fileSystem.readFileSync('../data/list.txt').toString().split('\n');
+		var languageCodeValid = validLanguageCodes.indexOf(languageCode);
+
+		if (languageCodeValid === -1){
+			return "Error: Not a valid language code"
```

Comment:
We may need to agree a consistent error message for invalid language case.

Reply:
Agreed, That would definitely help clean things up!